UnitTests/test_driver.cpp

```diff
@@ -0,0 +1,37 @@
+/*--- This tells Catch to provide a main().  This should only be done in
+ * one cpp file. ---*/
+#define CATCH_CONFIG_RUNNER
+#include "catch.hpp"
+
+#include "../../../Common/include/mpi_structure.hpp"
+#include "../../../Common/include/option_structure.hpp"
+
+int main(int argc, char *argv[]) {
+
+  /*--- Startup MPI, if supported ---*/
+#ifdef HAVE_MPI
+  int  buffsize;
+  char *buffptr;
+#ifdef HAVE_OMP
+  int provided;
+  SU2_MPI::Init_thread(&argc, &argv, MPI_THREAD_FUNNELED, &provided);
+#else
+  SU2_MPI::Init(&argc, &argv);
```

Comment:
Looks like the driver sets up MPI.. it would be great if we can also use MPI in the tests. Is that working?

Reply:
I haven't tested it myself, but that's the intention.

A lot of classes (like the solver classes) use MPI calls.  For example, some constructors set the MPI rank and size. So there's really no getting around MPI, even for non-MPI tests.